command/namespace_inspect.go

```diff
@@ -78,9 +78,13 @@ func (c *NamespaceInspectCommand) Run(args []string) int {
 		return 1
 	}
 
-	if len(possible) != 0 {
-		c.Ui.Error(fmt.Sprintf("Prefix matched multiple namespaces\n\n%s", formatNamespaces(possible)))
-		return 1
+	if len(possible) > 1 {
```

Comment:
Should you also check that ns == nil? Is it possible than ns is an exact match, but len(possible) is also greater than 1?